default/stringtables/en.txt

```diff
@@ -11984,8 +11984,8 @@ Confederation
 PLC_CONFEDERATION_DESC
 '''Planets of the empire are organized in a confederacy of independent systems.
 
-• Increases [[metertype METER_TARGET_INFLUENCE]] on colonies disconnected from the capital by [[value PLC_CONFEDERATION_SUPPLY_DISCONNECTED_INFLUENCE_FLAT]]
-• Increases [[metertype METER_TARGET_HAPPINESS]] on colonies disconnected from the capital and regional administrative centres by [[value DISCONNECTED_FROM_CAPITAL_AND_REGIONAL_ADMIN_STABILITY_PENALTY]]
+• Removes [[metertype METER_TARGET_INFLUENCE]] penalty on colonies disconnected from the capital
+• Removes [[metertype METER_TARGET_HAPPINESS]] penalty on colonies disconnected from the capital and regional administrative centres
```

Comment:
Maybe `• Removes [[metertype METER_TARGET_HAPPINESS]] penalty (baseline for most species) on colonies disconnected from the capital and regional administrative centres`

Reply:
ok will change it. There's probably a buildingtype for the capital too, imperial palace. I was planning on adding a modifier for planet stability (https://www.freeorion.org/forum/viewtopic.php?f=6&t=12214) and maybe one for planet suitability to the PR. Maybe still will. I assume you can selectively merge the commits, or I can remove contentious commits that don't pass.

Reply:
guess, I'll just leave any extra commits out for now pending any discussion that happens in the forum thread. Heading out, will fix the description when I get home and it'll be ready